packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -372,6 +391,22 @@ contract OPContractsManager is ISemver {
         return address(uint160(bytes20(bytes.concat(versionByte, first19Bytes))));
     }
 
+    /// @notice Helper method for computing a salt that's used in CREATE2 deployments.
+    /// Including the contract name ensures that the resultant address from CREATE2 is unique
+    /// across out smart conract system. For example, we deploy multiple proxy contracts
+    /// in a single deployment.
```

Comment:
(probably needs forge fmt)
```suggestion
    /// across our smart contract system. For example, we deploy multiple proxy contracts
    /// with the same bytecode from this contract, so they need different salts to avoid an address collision
```